lib/test-endpoint.sh

```diff
@@ -2,11 +2,23 @@
 
 unset DD_LOGS_VALID_ENDPOINT
 DATADOG_DIR="${DATADOG_DIR:-/home/vcap/app/datadog}"
+DD_EU_API_SITE="https://api.datadoghq.eu/api/"
+DD_US_API_SITE="https://api.datadoghq.com/api/"
+DD_API_SITE=$DD_US_API_SITE
+DD_USE_EU=false
 
-if [ -z $DD_LOGS_CONFIG_LOGS_DD_URL ]; then
+if [[ -n $DD_SITE ]] && [[ "$DD_SITE" == "datadoghq.eu" ]]; then
+  DD_USE_EU=true
+  DD_API_SITE=$DD_EU_API_SITE
+fi
+
+if [[ -z "$DD_LOGS_CONFIG_LOGS_DD_URL" ]]; then
   # Initialize to default value
-  if [ -n $DD_LOGS_CONFIG_DD_PORT -a -n $DD_LOGS_CONFIG_DD_URL ]; then
+  # If DD_SITE contains ".eu" use the EU endpoint, otherwise default to US
```

Comment:
This description doesn't seem to match what the source code below does, does he?

Reply:
🤦‍♂ Good catch, updated.